automation/deployment/deployer.py

```diff
@@ -48,23 +48,23 @@ class ExcecutionParams:
     def __init__(
         self,
         registry_url: str,
-        registry_secret_name: str = None,
-        chart_version: str = None,
-        mlrun_version: str = None,
-        override_mlrun_api_image: str = None,
-        override_mlrun_log_collector_image: str = None,
-        override_mlrun_ui_image: str = None,
-        override_jupyter_image: str = None,
+        registry_secret_name: typing.Optional[str] = None,
```

Comment:
For brevity, please import `Optional` in modules where it's missing.

Reply:
@assaf758 this change requires a lot of manual work or a clever way to detect imports across many modules.
I think it's ok for now, new code can choose to use `from typing import Optional` instead of `import typing`.

Reply:
Fair enough.
Please just fix the file conflict.